armi/reactor/components/componentParameters.py

```diff
@@ -239,6 +239,17 @@ def getHexHoledCircleParameterDefinitions():
     return pDefs
 
 
+def getFilletedHexagonParameterDefinitions():
+    """Return parameters for FilletedHexagon."""
+    pDefs = parameters.ParameterDefinitionCollection()
+    with pDefs.createBuilder(location=ParamLocation.AVERAGE, saveToDB=True) as pb:
+        pb.defParam(
+            "cornerR", units=units.CM, description="Radius of curvature of the corners"
+        )
+
+    return pDefs
+
```

Comment:
To be honest I don't fully understand this part of ARMI, so I might be misunderstanding. But why do we not want to also store other defining aspects of this shape, like `ip` and `op`? We store those things for the "normal" hexagon, but apparently not for the filleted hexagon or holed hexagon. Is this intentional?

Reply:
> But why do we not want to also store other defining aspects of this shape, like ip and op?

Don't worry! Those come from subclasses the hexagon.